### PR TITLE
Support `open` in swiftSyntaxBuilder

### PR DESCRIPTION
**Pull request by iMostfa**
This pull request adds support for the 'open' keyword as TokenSyntax in SwiftSyntaxBuilder
it's implemented as syntactic sugar based on `makeContextualKeyword ` 
and accessed in swift like this: TokenSyntax.public

I'm open to your feedback and suggestation to enhance my pull request 

greetings, Mostfa.

### CODE REVIEW

File: Sources/SwiftSyntaxBuilder/Tokens.swift.gyb [hunk 1]
```diff
@@ -62,4 +62,9 @@ public extension TokenSyntax {
   static var eof: TokenSyntax {
     SyntaxFactory.makeToken(.eof, presence: .present)
   }
+  /// The `open` token
```
Comment (ahoppen) [inline Sources/SwiftSyntaxBuilder/Tokens.swift.gyb:65]: One minor nitpick: Could you rephrase this to 
```suggestion
  /// The `open` contextual token
```

It should make it clear why it is not generated using `gyb`.

**Review by ahoppen — COMMENTED** [review]
Thanks. This looks good to me. Just one minor comment inline.

**Comment by iMostfa** [comment]
Thank you @ahoppen 
i've added your comment in Token.swift.gyb and the generated file

**Review by ahoppen — APPROVED** [review]

**Comment by ahoppen** [comment]
Thanks for the PR and welcome to SwiftSyntax development.
